test/elastic_record/index/documents_test.rb

```diff
@@ -151,9 +162,12 @@ def test_bulk_inheritance
         InheritedWidget.elastic_index.index_document '5', color: 'green'
 
         expected = [
-          {index: {_index: index.alias_name, _id: "5"}},
+          {index: {_index: index.alias_name, _id: "5", _type: '_doc'}},
```

Comment:
Perhaps have a helper for these expected index hashes that adds _type for Elasticsearch 6.